### DISCUSSION TITLE
User-level dashboard

### DISCUSSION THREAD

**Discussion by secsilm** [langfuse/langfuse#1794] [Ideas]
### Describe the feature or potential improvement

Currently, historical usage data cannot be viewed on the `Users` page like on the dashboard page. It is hoped that a user-level dashboard can be added.

**Comment by marcklingen** [comment]
are you asking for a chart on trace volume / user? I understand the value, open for contributions on this. I think it makes sense to add a usage chart to the profile of a single user
